prog/dftb+/prg_dftb/initprogram.F90

```diff
@@ -78,8 +78,18 @@ module initprogram
   use potentials
   use taggedoutput
   use formatout
+#:if WITH_GPU
+  use iso_c_binding, only :  c_int
+  use device_info
+#:endif
+
   implicit none
 
+#:if WITH_GPU
+  integer (c_int) :: ngpus
```

Comment:
The current magma library fortran wrappers for the actual calls seem to be integer not c_int